frontend/src/components/editor/cell/code/cell-editor.tsx

```diff
@@ -335,17 +336,26 @@ const CellEditorInternal = ({
     };
   }, [editorViewRef]);
 
-  const temporarilyShowCode = async () => {
+  const temporarilyShowCode = useCallback(async () => {
     if (hidden) {
       updateCellConfig({ cellId, config: { hide_code: false } });
       editorViewRef.current?.focus();
-      editorViewParentRef.current?.addEventListener(
+      editorViewRef.current?.contentDOM.addEventListener(
```

Comment:
Hmm, looks like my fix in #2508 got reverted. Is this intentional?

Reply:
Following up with a fix, `blur -> focusout`

Reply:
Yes sorry, I left a comment on that PR but forgot to put one here. And thanks, that did the trick!

Reply:
Well actually, it looks like you can't delete cells again...